.github/workflows/release.yml

```diff
@@ -107,3 +107,16 @@ jobs:
       with:
         gradle-executable: "./test_runner/gradlew"
         arguments: "-p test_runner publish -PGITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}"
+
+    - name: Get Latest Release notes and set as variables
+      run: |
+        testVar=$(awk '!NF{exit}1' release_notes.md)
+        echo "::set-env name=RELEASE_NOTES::$(echo $testVar)";
+
+    - name: Post Message of Flank Release TEST ONLY
+      uses: archive/github-actions-slack@master
```

Comment:
Should run only for release

Reply:
yes as far as i am aware it will only run for release. If you are concerned about the name its because its a test version for our internal slack as we dont have permissions for the bot in firebase. For now it will be internal to make sure it all works correctly.

Reply:
Later a new pull request with changes made to update the oauth & channel name for the firebase community

Reply:
Yeah I mean that there should be if check to prevent job running on snapshot builds
Something like:
`      if: startsWith(github.ref, 'refs/tags/v')`

Reply:
fixed 👍